target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-rutx11.dts

```diff
@@ -0,0 +1,210 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4018-rutx.dtsi"
+
+/ {
+	model = "Teltonika RUTX11";
+	compatible = "teltonika,rutx11";
+
+	memory {
+		device_type = "memory";
+		reg = <0x80000000 0x10000000>;
+	};
+	
+	aliases {
+		label-mac-device = &gmac0;
+	};
+
+	soc {
+
+		i2c_0: i2c@78b7000 {
+			stm32_io: stm32@74 {
+				#gpio-cells = <2>;
+				compatible = "tlt,stm32v1";
+				reg = <0x74>;
+				gpio-controller;
+				interrupt-parent = <&tlmm>;
+				interrupts = <5 2>;
+				interrupt-controller;
+				#interrupt-cells = <2>;
+			};
+		};
+
+		mdio@90000 {
+			status = "ok";
+			pinctrl-0 = <&mdio_pins>;
+			pinctrl-names = "default";
+			phy-reset-gpio = <&tlmm 62 0>;
+
+			ethphy4: ethernet-phy@4 {
+				qcom,fiber-enable;
+			};
+		};
+
+		pinctrl@1000000 {
+			mdio_pins: mdio_pinmux {
+				mux_1 {
+					pins = "gpio53";
+					function = "mdio";
+					bias-pull-up;
+				};
+				mux_2 {
+					pins = "gpio52";
+					function = "mdc";
+					bias-pull-up;
+				};
+			};
+		};
+
+		gpio_keys {
+			compatible = "gpio-keys";
+
+			reset {
+				label = "reset";
+				gpios = <&tlmm 4 1>;
+				linux,code = <0x198>;
+			};
+		};
+		
+		usb3: usb3@8af8800 {
+			dwc3@8a00000 {
+				snps,dis_u2_susphy_quirk;
+				snps,dis_u3_susphy_quirk;
+			};
+		};
+
+		usb2: usb2@60f8800 {
+			dwc3@6000000 {
+				snps,dis_u2_susphy_quirk;
+			};
+		};
+
+		gpio-export {
+			compatible = "gpio-export";
+			#size-cells = <0>;
+
+			gpio_modem_reset {
+				gpio-export,name = "modem_reset";
+				gpio-export,output = <0>;
+				gpios = <&stm32_io 21 GPIO_ACTIVE_HIGH>;
+			};
+
+			gpio_modem_power {
+				gpio-export,name = "modem_power";
+				gpio-export,output = <0>;
+				gpios = <&stm32_io 20 GPIO_ACTIVE_HIGH>;
+			};
+
+			gpio_modem2_reset {
+				gpio-export,name = "modem2_reset";
+				gpio-export,output = <0>;
+				gpios = <&stm32_io 13 GPIO_ACTIVE_HIGH>;
+			};
+
+			gpio_modem2_power {
+				gpio-export,name = "modem2_power";
+				gpio-export,output = <0>;
+				gpios = <&stm32_io 14 GPIO_ACTIVE_HIGH>;
+			};
+
+			gpio_sim_select {
+				gpio-export,name = "sim_sel";
+				gpio-export,output = <1>;
+				gpios = <&stm32_io 22 GPIO_ACTIVE_LOW>;
+			};
+
+			gpio_out_1 {
+				gpio-export,name = "gpio23";
+				gpio-export,output = <0>;
+				gpio-export,direction_may_change;
+				gpios = <&stm32_io 23 GPIO_ACTIVE_HIGH>;
+			};
+
+			gpio_in_1 {
+				gpio-export,name = "gpio24";
+				gpio-export,input = <0>;
+				gpio-export,direction_may_change;
+				gpios = <&stm32_io 24 GPIO_ACTIVE_LOW>;
+			};
+		};
+
+		leds {
+			compatible = "gpio-leds";
+
+			led_wifi2g {
+				label = "wifi_2g";
```

Comment:
LED labels normally follow the pattern color:function. This would be `green:wlan2g` and line 140 would be `green:wlan5g`. Similar for the other LEDs that follow.

Reply:
Fixed